jekyll/_cci2/introduction-to-yaml-configurations.adoc

```diff
@@ -144,11 +144,14 @@ YAML can be quite picky about indentations. You can use a http://yaml-online-par
 If you would like a more complex configuration tutorial, visit the <<config-intro#,Configuration Tutorial>> page. Before starting this tutorial, you should already have a CircleCI account set up, as you will follow along in the CircleCI web UI. You can also find a variety of configuration examples on the <<sample-config#, Sample Configuration>> page.
 
 [#visual-studio-code-extension]
-## Visual Studio Code extension
+## VS Code extension
```

Comment:
Should we include a relevant screenshot/capture here, or just try and keep most images on the overview page?